dev/com.ibm.ws.jmx/.classpath

```diff
@@ -4,5 +4,18 @@
 	<classpathentry kind="src" output="bin_test" path="test"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="src" path="/com.ibm.ws.kernel.service"/>
+	<classpathentry kind="lib" path="C:/Users/IBM_ADMIN/.m2/repository/org/osgi/org.osgi.service.event/1.3.1/org.osgi.service.event-1.3.1.jar" sourcepath="C:/Users/IBM_ADMIN/.m2/repository/org/osgi/org.osgi.service.event/1.3.1/org.osgi.service.event-1.3.1.jar">
```

Comment:
Should probably clean this up. It's pointing to a location on your local hard drive. After your refactoring could you just remove these additions to the .classpath file?

Reply:
Cleaned this up, about to push out to the branch